openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -105,7 +131,7 @@ def with_inline_css(html_without_css):
     """
     css_filepath = settings.NOTIFICATION_EMAIL_CSS
     if not css_filepath.startswith('/'):
-        css_filepath = finders.FileSystemFinder().find(settings.NOTIFICATION_EMAIL_CSS)
+        css_filepath = file_path_finder(settings.NOTIFICATION_EMAIL_CSS)
```

Comment:
There are different file finders e.g., `edx-notifications` is using `AppDirectoriesFinder` for finding files but we need to get files (credit notification html, css and logo) from static directory so I used `FileSystemFinder`.
So to make sure that we are using same finder for finding all files from static directory I create a separate `file_path_finder` method.